release/changes.md

```diff
@@ -0,0 +1 @@
+[FIX] Support build scan links to alternate GitHub URLs (such as GitHub Enterprise servers)
```

Comment:
The fix should describe the problem -- not the solution.

Reply:
Updated the fix to describe the problem being addressed